src/FolderConfig.cpp

```diff
@@ -0,0 +1,94 @@
+#include "FolderConfig.h"
+#include <QFile>
+#include <QDebug>
+#include<QDir>
+
+namespace QtDBMigrationNS {
+FolderConfig::FolderConfig(const QString &path): AbstractConfig(path)
+{
+}
+
+bool FolderConfig::load()
```

Comment:
Can you split this function to make it more readable?

Reply:
Function split.